oggm/tests/test_utils.py

```diff
@@ -1205,6 +1210,24 @@ def test_elev_bands_run(self):
             # We can't create this because the glacier dir is mini
             tasks.init_present_time_glacier(gdir)
 
+        # Summary
+        hf = os.path.join(odir, 'RGI61', bstr, 'L5', 'summary',
+                          'historical_run_output_11.nc')
+        ef = os.path.join(odir, 'RGI61', bstr, 'L5', 'summary',
+                          'historical_run_output_extended_11.nc')
+        sf = os.path.join(odir, 'RGI61', bstr, 'L5', 'summary',
+                          'dynamic_spinup_run_output_11.nc')
+
+        import matplotlib.pyplot as plt
+        with xr.open_dataset(hf) as dsh:
+            dsh = dsh.load()
+        with xr.open_dataset(ef) as dse:
+            dse = dse.load()
+        with xr.open_dataset(sf) as dss:
+            dss = dss.load()
+        assert len(dss.time) > 1
```

Comment:
@pat-schmitt I rebased your branch and added a commit. You'll need to force pull, but I think I didn't brake anything.

Can you please have a look at this test and what I'm doing wrong? the output only has one year, and I didn't see any error message pointing me to the right direction here